apps/api/src/campaign-news/campaign-news.service.ts

```diff
@@ -101,7 +101,7 @@ export class CampaignNewsService {
         campaignid: campaign.id,
         template_data: {
           'campaign.name': campaign?.title,
-          'campaign.target-amount': campaign?.targetAmount || 0,
+          'campaign.target-amount': (campaign?.targetAmount && campaign?.targetAmount / 100) || 0,
```

Comment:
This is somewhat difficult to read. 
I wonder if there is a better way to get the campaign target.